docs/docs-content/getting-started/cluster-profiles.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: "Cluster Profiles"
+title: "Cluster Profiles"
+description: "Spectro Cloud Palette Cluster Profiles"
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["getting-started"]
+---
+
+Cluster profiles are the declarative, full-stack models that Palette follows when it provisions, scales, and maintains
+your clusters. Cluster profiles are composed of layers using packs, Helm charts, Zarf packages, or cluster manifests to
+meet specific types of workloads on your Palette cluster deployments. You can create as many profiles as needed for your
+workloads.
+
+Once you create your profiles, the provide you with a repeatable deployment process for all of your development and
```

Comment:
I think something got dropped in this sentence by accident 🙈

Reply:
Oops! Fixed this up now.